healthcheck/healthcheck.go

```diff
@@ -0,0 +1,37 @@
+package healthcheck
+
+import (
+	"github.com/cloudwan/gohan/db"
+	"github.com/cloudwan/gohan/db/transaction"
+	"github.com/cloudwan/gohan/sync"
+)
+
+type Healthchecker interface {
+	DoHealthcheck() bool
+}
+
+type Healthcheck struct {
+	DataStore db.DB
+	Etcd      sync.Sync
+}
+
+func (healthcheck Healthcheck) DoHealthcheck() bool {
+	err1 := db.WithinTx(healthcheck.DataStore, func(transaction transaction.Transaction) error {
+		return nil
+	})
+	_, err2 := healthcheck.Etcd.Fetch("/gohan/cluster/lock/state_watch")
```

Comment:
this key may not be the right choice: if the lock is lost, the key will disappear from ETCD and all healthchecks will return 503.
I suggest just `/gohan` instead

Reply:
using `/gohan` now